.github/scripts/fetch_external_sources.sh

```diff
@@ -30,22 +58,27 @@ fetch_for_file() {
     if [ -n "${line}" ] && [[ "${line}" == *.md ]] || [[ "${line}" == *.png ]]; then
       # If the file exists do nothing, otherwise pull it in from github
       local file_to_fetch=${file_dir}/${line}
-      if ! ls "${file_to_fetch}" > /dev/null 2>&1; then
+      if [ "${FORCE}" = true ]; then
+        echo "Force option enabled. Trying to fetch '${line}' from github"
+      elif ! ls "${file_to_fetch}" > /dev/null 2>&1; then
         echo "${line} does not exist. Trying to fetch it from github"
-        local outputdir=$(dirname ${file_to_fetch})
-        mkdir -p ${outputdir}  # make the directory for the output
-
-        # Try a few github organizations
-        for org in key4hep HEP-FCC AIDASoft iLCSoft; do
-          echo "Trying to fetch from github organization: '${org}'"
-          if try_fetch ${org} ${line} ${file_dir}; then
-            echo "Fetched succesfully from organization '${org}'"
-            break
-          fi
-        done
+      else
+        continue
       fi
 
-      # Check again if we hav succesfully fetched the file
+      local outputdir=$(dirname ${file_to_fetch})
+      mkdir -p ${outputdir}  # make the directory for the output
+
+      # Try a few github organizations
+      for org in key4hep HEP-FCC AIDASoft iLCSoft; do
```

Comment:
Looking at this again, I think it should be fairly straight forward to make it possible to get content from (user) forks here by just pre-pending this list with more "organizations" (aka user names) where we could also take input via CL arguments.

The only downside to the dumb approach then is that we will give precedence to all user forks, when users potentially just want to give precedence to one repository.

Also we would need to make `try_fetch` actually consider more branches than `main` and `master`.

Maybe it's not so straight forward after all.

Reply:
Yeah, that was my concern as well. If we want an override here it needs to be package specific. This is already a step forward as it is.